### DISCUSSION TITLE
Server only loads one project

### DISCUSSION THREAD

**Discussion by Gabrielle-Tobermann** [nss-evening-cohort-14/evening-client-side#142] [Help Me!] [answered]
Gabby Tobermann 

Other tickets? I found old tickets on localhost not functioning but this is a different problem. 

Issue: I was working on the array methods but I cannot check my work. When I start my server, http://127.0.0.1:8000 only loads my Gitsub team project. I cannot load any other project but this one. I've tried to hard refresh but that doesn't change anything. 

Errors: no errors. 

Google links: https://stackoverflow.com/questions/37352603/localhost-refused-to-connect-error-in-visual-studio
This one talks about delete a file inside .vs folder, but I cannot find that file on my machine (I've used mac all my life and I am still getting used to using Windows...) 
https://stackoverflow.com/questions/32190881/how-to-solve-err-connection-refused-when-trying-to-connect-to-localhost-running
https://stackoverflow.com/questions/33524826/localhost-not-working-in-chrome-127-0-0-1-does-work
A lot of these suggestions on the links I found involve deleting files or changing ports, which I have not tried because I don't fully understand what's going on. 

What I have tried: Serving my pet-adoption project, but it still loads the Gitsub team project. I tried serving directly from vs code (I usually have to serve from my windows terminal) by clicking on "start debugging", choosing chrome. In this case, the address is "http://localhost:8080" but this doesn't load anything, it just tells me localhost refused to connect. 
I've also tried closing all apps and restarting the process, but nothing has changed.

**Comment by drteresavasquez** [comment, marked as answer]
@Gabrielle-Tobermann did you stop your server for the gitSub project? Also, when you hard refresh, is your inspector open?

**Reply by Gabrielle-Tobermann** [reply]
Yes. I reopened the Gitsub project, served and then stopped it. And yes, my inspector was open, there were no errors. Also, when working on a project which has the webpack template, I have no problem running npm start, the page loads just fine.

**Reply by drteresavasquez** [reply]
So are you running the `hs -o` command for the array methods? Also make sure you have a html file that connects everything.

**Reply by drteresavasquez** [reply]
Where is the link to your code?

**Reply by Gabrielle-Tobermann** [reply]
Actually it looks like restarting my computer did the trick, it's working now! Thanks for the help though :)

**Reply by drteresavasquez** [reply]
@Gabrielle-Tobermann go ahead and post your fix in a main comment so we can mark this as answered. Thanks!
